### PR TITLE
feat: add password_hash to signup parameters

### PR DESCRIPTION
**Pull request by penguinuux**
Added the key to request body example.

### CODE REVIEW

**Review by luizpcardoso — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -28,13 +32,14 @@ Register a new user.
...
-| cep       | string   | User location cep number |
-| city      | string   | User city location       |
+| **Field**     | **Type** | **Description**          |
+| ------------- | -------- | ------------------------ |
+| name          | string   | User Name                |
+| email         | string   | User email               |
+| phone         | string   | User phone number        |
+| cep           | string   | User location cep number |
+| city          | string   | User city location       |
+| password_hash | string   | User password            |
```
Comment (marcelosj3) [inline README.md]: Shouldn't it be just password? the password_hash is something we create in our database.

File: README.md [hunk 2]
```diff
@@ -44,7 +49,8 @@ Register a new user.
   "email": "d.motta@gmail.com",
   "phone": "24976520981",
   "cep": "25651-079",
-  "city": "Petrópolis"
+  "city": "Petrópolis",
+  "password_hash": "uhtw86#b"
```
Comment (marcelosj3) [inline README.md]: Same as above.

**Review by marcelosj3 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (penguinuux) [reply README.md]: Done.

File: README.md (hunk 2 above)
Comment (penguinuux) [reply README.md]: Done.

**Review by luizpcardoso — APPROVED** [review]

**Review by marcelosj3 — APPROVED** [review]

**Pull request merged**